ViewModel/ImpactAreaScenario/Results/PerformanceAssuranceOfThresholdVM.cs

```diff
@@ -18,14 +18,14 @@ private void LoadData(ImpactAreaScenarioResults iasResult, List<ThresholdComboIt
             {
                 int thresholdKey = thresholdComboItems[i].Metric.ThresholdID;
                 List<IPerformanceRowItem> rows = new List<IPerformanceRowItem>();
-                List<double> xVals = new List<double>(){ .1, .04, .02, .01, .005, .002};
+                List<double> xVals = new List<double>(){ .1, .04, .02, .01, .004, .002};//exceedance probabilities
                 foreach (double xVal in xVals)
                 {
-                    double exceedanceProb = 1.0 - xVal;
+                    double nonExceedanceProb = 1.0 - xVal;
                     try
                     {
-                        double yVal = iasResult.AssuranceOfEvent(thresholdKey, xVal);
-                        rows.Add(new PerformanceFrequencyRowItem(xVal, yVal));
+                        double yVal = iasResult.AssuranceOfEvent(thresholdKey, nonExceedanceProb);
+                        rows.Add(new PerformanceFrequencyRowItem(xVal, yVal)); //should this be nonExceedanceProb too?
```

Comment:
Best to put comment above line, and if it is a question, best to add "TODO" at the beginning so that we can more easily address it later.

Reply:
fixed

Reply:
and I think this was the fix that was giving NaNs ... similar problem